x-pack/plugins/upgrade_assistant/server/lib/es_deprecation_logging_apis.ts

```diff
@@ -20,7 +17,8 @@ export async function getDeprecationLoggingStatus(
   });
 
   return {
-    isEnabled: isDeprecationLoggingEnabled(response),
+    isEnabled: isClusterDeprecationLoggingEnabled(response),
+    isLoggerDeprecationEnabled: isDeprecationLoggingEnabled(response),
```

Comment:
Can we ripple the name changes downstream? I find it easier to trace code when naming patterns are maintained across functions, returned values, and so on. This would become:

```js
    isDeprecationLogIndexingEnabled: isDeprecationLogIndexingEnabled(response),
    isDeprecationLoggingEnabled: isDeprecationLoggingEnabled(response),
```